src/EnergyPlus/SolarShading.cc

```diff
@@ -10444,8 +10455,11 @@ void SkyDifSolarShading(EnergyPlusData &state)
             if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor != -1) {
                 state.dataSurface->Surface(SurfNum).ViewFactorSkyIR *= state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor;
             }
-            if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).GroundViewFactor != -1) {
-                state.dataSurface->Surface(SurfNum).ViewFactorGroundIR *= state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).GroundViewFactor;
+        }
+        if (state.dataSurface->IsSurfPropertyGndSurfacesDefined(SurfNum)) {
+            int GndSurfsNum = state.dataSurface->GroundSurfsPropertyNum(SurfNum);
+            if (state.dataSurface->GroundSurfsProperty(GndSurfsNum).IsGroundViewFactorSet) {
+                state.dataSurface->Surface(SurfNum).ViewFactorGroundIR *= state.dataSurface->GroundSurfsProperty(GndSurfsNum).SurfsViewFactorSum;
```

Comment:
The ground view factor obtained from the new object is set here if the ground view factor input field is blank.